interface/orders/types_edit.php

```diff
@@ -72,81 +71,152 @@ function recursiveDelete($typeid)
         "procedure_type_id = '$typeid'");
 }
 
+
 ?>
+<!DOCTYPE html>
 <html>
 <head>
-    <?php Header::setupHeader(['opener']); ?>
-    <title><?php echo $typeid ? xlt('Edit') : xlt('Add New'); ?><?php echo xlt('Order/Result Type'); ?></title>
+    <?php Header::setupHeader(['datetime-picker', 'jquery-ui', 'jquery-ui-base']);?>
+<script type="text/javascript"
+    src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
+<title><?php echo $typeid ? xlt('Edit') : xlt('Add New'); ?> <?php echo xlt('Order/Result Type'); ?></title>
 
-    <script language="JavaScript">
 
-        <?php require($GLOBALS['srcdir'] . "/restoreSession.php"); ?>
+<style>
+td {
+    font-size: 10pt;
+}
 
-        // The name of the form field for find-code popup results.
-        var rcvarname;
+.inputtext {
+    padding-left: 2px;
+    padding-right: 2px;
+}
 
-        // This is for callback by the find-code popup.
-        // Appends to or erases the current list of related codes.
-        function set_related(codetype, code, selector, codedesc) {
-            var f = document.forms[0];
-            var s = f[rcvarname].value;
-            if (code) {
-                if (s.length > 0) s += ';';
-                s += codetype + ':' + code;
-            } else {
-                s = '';
-            }
-            f[rcvarname].value = s;
-        }
+.button {
+    font-family: sans-serif;
+    font-size: 9pt;
+    font-weight: bold;
+}
 
-        // This is for callback by the find-code popup.
-        // Returns the array of currently selected codes with each element in codetype:code format.
-        function get_related() {
-            return document.forms[0][rcvarname].value.split(';');
-        }
+.ordonly {
+    
+}
 
-        // This is for callback by the find-code popup.
-        // Deletes the specified codetype:code from the currently selected list.
-        function del_related(s) {
-            my_del_related(s, document.forms[0][rcvarname], false);
-        }
+.resonly {
+    
+}
+.label-div > a {
+    display:none;
+}
+.label-div:hover > a {
+   display:inline-block; 
+}
+div[id$="_info"] {
+    background: #F7FAB3;
+    padding: 20px;
+    margin: 10px 15px 0px 15px;
+}
+div[id$="_info"] > a {
+    margin-left:10px;
+}
+@media only screen {
+        fieldset > [class*="col-"] {
+        width: 100%;
+        text-align:left!Important;
+    }
+}
+/*@media only screen and (max-width: 450px) {
+        #button-container > [class*="col-"] {
+        width: 100%;
+        text-align:left!Important;
+    }*/
+</style>
+
+<script type="text/javascript" src="../../library/topdialog.js"></script>
+<script type="text/javascript" src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
```

Comment:
remove above 2 lines and add `topdialog` to the above setupHeader call above
(dialog.js is added automatically already)

Reply:
Done